database/seeds/DatabaseSeeder.php

```diff
@@ -12,6 +13,12 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        $this->call(QuestionTableSeeder::class);
+        foreach(range(1, 20) as $level) {
+            $question = factory(Question::class)->create(['level' => $level]);
+
+            factory(QuestionAttachment::class)->create([
+                'question_id' => $question->id
+            ]);
+        }
```

Comment:
make separate seeder class `QuestionTableSeeder`  and call it from this class like:
```php
public function run() 
{
    $this->call(QuestionTableSeeder::class);
}
```